ibc/modules/src/core/ics23_commitment/commitment.rs

```diff
@@ -30,7 +30,7 @@ pub struct CommitmentRoot {
 
 impl fmt::Debug for CommitmentRoot {
 	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-		let hex = Hex::upper_case().encode_to_string(&self.bytes).unwrap();
+		let hex = Hex::upper_case().encode_to_string(&self.bytes).unwrap_or_default();
```

Comment:
why is returning the default a good option?

Reply:
I know it's debug, but just wondering. It's a way to signal that there's nothing on the stderr/stdout?